### ISSUE TITLE
no training

### ISSUE THREAD

**Issue by guanrenxue** [haeyunlee/LSS-Net#3]
I am very interested in your work. I used the original data set and generated the corresponding train.h5 and val.h5, but did not train. How can I solve it?
error:
  /home/ubuntu/anaconda3/envs/lss/bin/python3.6 /home/whu/LSS/train.py
Loading dataset ...

# of training samples: 10000

Success to load a pretrained weight
2998
learning rate 0.000100
Keep staying at "learning rate 0.000100", no training, looking forward to your reply.

**Comment by haeyunlee** [comment]
I'm sorry for the late reply.

I don't know exactly what the problem is just by looking at the writing.

Is the train.h5 file created properly?
